package.json

```diff
@@ -13,9 +13,12 @@
     "icons": "yarn --cwd packages/odyssey-icons svgo",
     "lerna-version": "lerna version --no-git-tag-version --force-publish",
     "prelerna-publish": "lerna run build",
-    "lerna-publish": "lerna publish from-package --no-push --force-publish --no-verify-access --no-verify-registry"
+    "lerna-publish": "lerna publish from-package --no-push --force-publish --no-verify-access --no-verify-registry",
+    "lint": "eslint '**/*.@(js|vue)' --ignore-pattern node_modules/ && stylelint ./*.scss", 
```

Comment:
I feel a little weird about how I chose to specify the file extension in the glob instead of the eslint flag for extension, but the latter wasn't working as expected. This does!

Reply:
I'd assume that the plugin should provide the necessary overrides to run this on `js` and `vue` extensions. We should be able to do the following to simply this call:

1. Create an `.eslintignore` file that has `node_modules` and `dist` (I assume this is the VuePress build output) added to it
2. Update this script to run on all packages in this mono-repo:
   ```diff
   -"eslint '**/*.@(js|vue)' --ignore-pattern node_modules/ && stylelint ./*.scss"
   +"eslint . && stylelint ./*.scss"
   ```

Reply:
I had tried the approach you are outlining here earlier and got the following result:

![image](https://user-images.githubusercontent.com/62259644/96942647-8edfb500-148a-11eb-998b-fcdd4de265a0.png)

Note: there is an error in the `.vue` file above.

When I add the  `--ext` flag to include `.vue` files it works fine

![image](https://user-images.githubusercontent.com/62259644/96943014-a8353100-148b-11eb-9c3a-b215e7f5b7df.png)

Pushing it up as is, let me know if you think this requires further changes.

Reply:
This is likely because `lint` is being run from the package instead of the root - but I'm OK with this for now.